Feature/eulerian storm track

jeyavinoth opened the pull request:
I am the Research Tech working with Dr. Jimmy Booth at City College of New York, CUNY. This is the one of two PODs that we will submit. This POD is the Eulerian Storm Track. 

The files I have modified outside the eulerian_storm_track folder are the following: 
1. src/default_jj.jsonc - my settings file to run only our POD for the specified years
2. src/conda/env_eulerian_storm_track.yml - necessary packages to run the POD 

Thanks, 
JJ

jkrasting commented:
Thanks for the initial POD contribution @jeyavinoth!

jkrasting commented on diagnostics/eulerian_storm_track/eulerian_storm_track_util.py:
```diff
@@ -0,0 +1,177 @@
+      time_ind = ((dates_year == i_year) & (dates_month == 9)) | ((dates_year == i_year) & (dates_month == 10)) | ((dates_year == i_year) & (dates_month == 11)) 
+
+    with warnings.catch_warnings():
+      warnings.simplefilter("ignore", category=RuntimeWarning)
```
Is this to catch warnings issued by operating on arrays that contain NaNs?

jeyavinoth replied:
Yes it is.

jkrasting commented on diagnostics/eulerian_storm_track/notes:
```diff
@@ -0,0 +1 @@
+time for daily is "days since start year"
```
What is this purpose of this notes file?

jeyavinoth replied:
It was an internal file, I removed it.

jkrasting reviewed (changes requested):
Thanks for your contribution @jeyavinoth and @jfbooth!  @tsjackson-noaa -- I approve moving forward with this PR inside the NOAA network.

@jeyavinoth - we need a few updates:

The POD requires an *.rst formatted documentation file
Please delete the `remove` subdirectory if it is not needed
Is `src/default_jj.jsonc` needed?  If not, please remove
Please remove `diagnostics/eulerian_storm_track/eulerian_storm_track.html.bak` 

There are also two separate conda environments here.  Is there a way to consolidate to one, or better yet, @tsjackson-noaa is there a way to merge these into the base environment?

@tsjackson-noaa will likely have some more edits and suggestions, but we look forward to working with you to merge this in.

@bitterbark, @yihungkuo, @ahmedfiaz, @dneelin - please also take a look at let us know if you see anything upfront that could be an issue.

jeyavinoth commented:
> Thanks for your contribution @jeyavinoth and @jfbooth! @tsjackson-noaa -- I approve moving forward with this PR inside the NOAA network.
> 
> @jeyavinoth - we need a few updates:
> 
> * The POD requires an *.rst formatted documentation file
> * Please delete the `remove` subdirectory if it is not needed
> * Is `src/default_jj.jsonc` needed?  If not, please remove
> * Please remove `diagnostics/eulerian_storm_track/eulerian_storm_track.html.bak`
> 
> There are also two separate conda environments here. Is there a way to consolidate to one, or better yet, @tsjackson-noaa is there a way to merge these into the base environment?
> 
> @tsjackson-noaa will likely have some more edits and suggestions, but we look forward to working with you to merge this in.
> 
> @bitterbark, @yihungkuo, @ahmedfiaz, @dneelin - please also take a look at let us know if you see anything upfront that could be an issue.

@jkrasting - I have made the following changes: 
* I added *.rst formatted documentation file. 
* I removed the `remove` subdirectory
* I removed `src/default_jj.jsonc` and `diagnostics/eulerian_strom_track/eulerian_storm_track.html.bak`
* In addition I removed `diagnostics/eulerian_strom_track/test.py`, `diagnostics/eulerian_strom_track/example.py` and `diagnostics/eulerian_strom_track/plotter_cartopy.py`. I was not using these files. 

I pushed the changes to my github fork. Is that enough for you to access the changes?
